### ISSUE TITLE
Saving as a draft in PS360 using OF 

### ISSUE THREAD

**Issue by zafar2zafar** [flexpeace/DI-Testing#22]
If you save report on PS360 as draft while reporting a case in real OF environment. Then you need a way to resume that. Need to discuss.

**Comment by avezrizvi** [comment]
Please close-out if validated fix.

**Comment by fareedpax** [comment]
No...I wouldnt close it yet. There is an educational piece that  we need to
discuss  on this..It doesnt require any build /change request.
